README.md

```diff
@@ -160,7 +177,7 @@ All of the dependencies can be found in the `build-unix.md` file located in the
 - Build requirements:
 
 ```
-sudo apt install build-essential libtool autotools-dev automake pkg-config bsdmainutils
+sudo apt install build-essential libtool autotools-dev automake pkg-config bsdmainutils python3
```

Comment:
doesn't raspbian ship with python3, if not why would it be a build requirement?

Reply:
I think python3 comes pre-installed with raspian, but someone may be using a different OS that doesn't come with python3. Unless we're only making a guide for raspian?

Also, python3 is included in the bitcoin-core build-unix.md file here: https://github.com/bitcoin/bitcoin/blob/master/doc/build-unix.md#linux-distribution-specific-instructions